tests/behaviour/concept/type/thingtype/thing_type_steps.py

```diff
@@ -67,6 +67,20 @@ def step_impl(context: Context, root_label: RootLabel, type_label: str, get_labe
     assert_that(context.get_thing_type(root_label, type_label).as_remote(context.tx()).get_label().name(), is_(get_label))
 
 
+@step("{root_label:RootLabel}({type_label}) set abstract: {is_abstract}; throws exception")
+def step_impl(context: Context, root_label: RootLabel, type_label: str, is_abstract):
```

Comment:
As a consequence of updating `@vaticle_typedb_behaviour` to the latest version, we need to implement this newly defined step.